<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesPage</class>
 <widget class="QWidget" name="PreferencesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="mShowIndicesBox">
     <property name="text">
      <string>Show indices</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="mSessionLimitLabel">
       <property name="text">
        <string>Sessions limit:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="mCountSessionsSpin"/>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mSessionDirLabel">
       <property name="text">
        <string>Session directory:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="mSessionDirectoryEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="mSessionDirectoryButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../visual_db.qrc">
         <normaloff>:/img/folder.png</normaloff>:/img/folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="mNewTabAutoSwitchBox">
     <property name="text">
      <string>Automatically switch to a new tab</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mLoadLastSessionBox">
     <property name="text">
      <string>Automatically load last session on start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../visual_db.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mSessionDirectoryButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesPage</receiver>
   <slot>folder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>folder()</slot>
 </slots>
</ui>
